security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -72,7 +91,7 @@ type SecretCache struct {
 	secretChangedCount uint64
 
 	// callback function to invoke when detecting secret change.
-	notifyCallback func(string, *model.SecretItem) error
+	notifyCallback func(string /*proxy ID*/, string /*resource Name*/, *model.SecretItem) error
```

Comment:
make this self-documenting by naming the parameters:
```go
	notifyCallback func(proxyID string, resourceName string, secret *model.SecretItem) error
```

Reply:
updated.